soda/src/main/java/it/okkam/opendata/models/soda/SodaClassification.java

```diff
@@ -50,4 +51,12 @@ public void setDomain_metadata(Map<String, String>[] domain_metadata) {
     this.domain_metadata = domain_metadata;
   }
 
+  public Map<String, String>[] getDomain_private_metadata() {
+    return domain_private_metadata;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') it.okkam.opendata.models.soda.SodaClassification.getDomain_private_metadata() may expose internal representation by returning SodaClassification.domain_private_metadata [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://dev.okkam.it/sonar/coding_rules#rule_key=findbugs%3AEI_EXPOSE_REP)